tests/testsuite/rust_version.rs

```diff
@@ -1210,10 +1234,13 @@ fn report_rust_versions() {
         .with_stderr_data(str![[r#"
 [UPDATING] `dummy-registry` index
 [LOCKING] 9 packages to latest Rust 1.60.0 compatible versions
-[ADDING] dep-only-high-incompatible v1.75.0 (requires Rust 1.75.0)
-[ADDING] dep-only-low-incompatible v1.75.0 (requires Rust 1.75.0)
+[ADDING] dep-only-high-compatible v1.55.0 (available: v1.65.0)
+[ADDING] dep-only-high-incompatible v1.55.0 (available: v1.75.0, requires Rust 1.75.0)
+[ADDING] dep-only-low-incompatible v1.55.0 (available: v1.75.0, requires Rust 1.75.0)
+[ADDING] dep-only-unset-compatible v1.55.0 (available: v1.75.0)
 [ADDING] dep-only-unset-incompatible v1.2345.0 (requires Rust 1.2345.0)
 [ADDING] dep-shared-incompatible v1.75.0 (requires Rust 1.75.0)
+[NOTE] 2 packages may have a higher, compatible version. To update them, run `cargo update <name> --precise <version>
```

Comment:
For me, this message is vague to the point of being unhelpful.

Reply:
In the case where we're *already* showing verbose messages about the individual crates, I agree. Could we show it only if there are messages about possible upgrades that we're currently hiding because of the verbosity level?

Reply:
Thats sounding like the message we already have
```
"pass `--verbose` to see {unchanged_behind} unchanged dependencies behind latest"
```